src/Board.js

```diff
@@ -160,7 +173,25 @@ function Board({ answer }) {
   function handlePianoPress(note) {
     handleKeyDown({ key: note });
   }
-
+  function shareResults() {
+    let stat = gameWon ? guess.join("").length / 6 : "X";
+    let beginText = `Musical Wordle - '${answer["song"]}' ${stat}/${guess.length}\n`;
+    navigator.clipboard
+      .writeText(
+        beginText +
+          shareOutput
+            .map((row) => {
+              return row.join("");
+            })
+            .join("\n")
+      )
+      .then(() => {
+        alert("Copied results to clipboard");
```

Comment:
Is there a way to avoid using `alert` here? There is an accessibility/best practice issue here, especially since IIRC Google et al. said they were going to block `alert`. 

In any event, we should add a TODO comment and create an issue/ticket.

Reply:
Agree -- maybe put a TODO comment here and file an issue?